RouteLocation.java

```diff
@@ -5,27 +5,38 @@
  */
 package kbs2;
 
+import java.util.ArrayList;
+
 /**
  *
  * @author Niek J Nijland
  */
 public class RouteLocation {
-    private int number;
+    private int Number;
```

Comment:
no upercase property names
```suggestion
    private int number;
```